.spectral.yml

```diff
@@ -37,6 +37,23 @@ rules:
 
   info-description: error
 
+  entur-info-metadata-id:
```

Comment:
Denne fila har en del kommentarer som tilsvarer seksjoner i `guidelines.md`. Siden du nå la på en helt ny seksjon for Entur Metadata midt inni, referer de nå til feil tall. Kanskje vi heller kunne putta "Entur Metadata" under seksjon 2?

Kunne samtidig også fikset opp på strukturen her sånn at alle reglene ligger under riktig seksjon.